docker-compose.yaml

```diff
@@ -0,0 +1,19 @@
+version: "3"
```

Comment:
I've added a docker-compose.yaml file which will build the container from the Dockerfile, but also run a selenium standalone container pulled from DockerHub.  This means we don't need to worry about configuring selenium and we can just consume the service which will be automatically set in the same network.

Running `docker-compose up` will run both containers together, run the acceptance tests then shut down the containers.

If you make a change to the code run `docker-compose build` first to recreate the acceptance test container.